babel.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (api) {
   }
 
   return {
+    sourceType: 'unambiguous',
```

Comment:
The language client is exported as CommonJS and I needed to do this to make it work. I'm not sure if this is the right solution though.

Reply:
@SleeplessByte?

Reply:
If someone can detail the issue, maybe 🗡️ 🥇

Reply:
We use this package https://github.com/TypeFox/vscode-ws-jsonrpc. But it's exported as CommonJS and we need this line in order to be able to use it. I'm actually not sure if I'm correct in this. This is how I just understood things.